python/vowpalwabbit/sklearn_vw.py

```diff
@@ -117,115 +120,192 @@ def __init__(self,
                  inpass=None,
                  meanfield=None,
                  multitask=None):
-        """VW model constructor, exposing all supported parameters to keep sklearn happy
+        r"""
```

Comment:
Why move the short summary to a new line?

Reply:
Sorry, it can be reverted to original one.